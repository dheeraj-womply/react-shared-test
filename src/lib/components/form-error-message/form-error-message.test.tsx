import renderer from "react-test-renderer";
import React from "react";
import { FormErrorMessage } from "./index";

test('Form Error Message are OK', () => {
    const component = renderer.create(
        <FormErrorMessage
            errorMessage="Error message"
        />
    )
    let tree = component.toJSON()
    expect(tree).toMatchSnapshot()
})
