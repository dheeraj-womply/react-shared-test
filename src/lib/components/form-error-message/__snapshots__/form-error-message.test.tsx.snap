// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form Error Message are OK 1`] = `
<div
  className="sc-AxjAm dntfnm"
  data-testid="form-error-message"
>
  <span>
    <svg
      aria-hidden={true}
      className="MuiSvgIcon-root sc-AxirZ fUiCmG"
      focusable="false"
      viewBox="0 0 24 24"
    >
      <path
        d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm1 15h-2v-2h2v2zm0-4h-2V7h2v6z"
      />
    </svg>
  </span>
  <span>
    Error message
  </span>
</div>
`;
